suite-native/graph/src/components/Graph.tsx

```diff
@@ -52,34 +63,63 @@ const graphStyle = prepareNativeStyle<GraphStyleProps>((_, { loading, error }) =
     opacity: loading || !!error ? 0.1 : 1,
 }));
 
-export const emptyGraphPoint: EnhancedGraphPointWithCryptoBalance = {
+export const emptyGraphPoint: FiatGraphPointWithCryptoBalance = {
     value: 0,
     date: new Date(0),
     cryptoBalance: '0',
-    originalDate: new Date(),
 };
 
-const emptyPoints: EnhancedGraphPointWithCryptoBalance[] = [
+const emptyPoints: FiatGraphPointWithCryptoBalance[] = [
     { ...emptyGraphPoint },
     { ...emptyGraphPoint, date: new Date(1) },
 ];
 
 // to avoid overflows from the screen
 const clampAxisLabels = (value: number) => N.clamp(value, 5, MAX_CLAMP_VALUE);
 
-export const Graph = <TGraphPoint extends EnhancedGraphPoint>({
+export const Graph = <TGraphPoint extends FiatGraphPoint>({
     onPointSelected,
     onGestureEnd,
     points = [],
     loading = false,
     animated = true,
     onTryAgain,
     error,
+    events,
 }: GraphProps<TGraphPoint>) => {
     const {
         applyStyle,
         utils: { colors },
     } = useNativeStyles();
+
+    const lastReferencedEvent = useRef<TransactionEventTooltipProps | null>(null);
+
+    const [activeTransactionEvent, setActiveTransactionEvent] =
+        useState<TransactionEventTooltipProps | null>(null);
+
+    const handleActiveTransactionEvent = useCallback(
+        (
+            newValue: TransactionEventTooltipProps | null,
+            cords: { eventX: number; eventY: number },
```

Comment:
just go with coordinates IMO. Not worth saving 2ms reading couple more characters over having a clear name